src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -410,6 +414,7 @@ def from_json(cls, json_str):
         log_analytics_resource_id='https://api.loganalytics.azure.cn',
         app_insights_telemetry_channel_resource_id='https://dc.applicationinsights.azure.cn/v2/track',
         synapse_analytics_resource_id='https://dev.azuresynapse.net'),
+        portal_uri='https://portal.azure.cn',
```

Comment:
this will need to register to azure metadata url also. let me share you the process.